scripts/builtin/msvm.dml

```diff
@@ -19,10 +19,20 @@
 #
 #-------------------------------------------------------------
 
-# Implements builtin multiclass SVM with squared slack variables, 
-# learns one-against-the-rest binary-class classifiers by making a function call to l2SVM
+* [`msvm`-Function](#mvsm-function)
```

Comment:
Changes are correct but to be done the [different file](https://github.com/apache/systemml/blob/master/dev/docs/builtins-reference.md)
🌵 😅